packages/react-devtools/CHANGELOG.md

```diff
@@ -2,6 +2,19 @@
 
 <!-- RELEASE_SCRIPT_TOKEN -->
 
+## 4.19.0 (September 28, 2021)
+
+#### Features
+* Don't patch console during first render in strict mode ([lunaruan](https://github.com/lunaruan) in [#22308](https://github.com/facebook/react/pull/22308))
+* Show which hook indicies changed when profiling for all builds ([bvaughn](https://github.com/bvaughn) in [#22365](https://github.com/facebook/react/pull/22365))
+* Display actual ReactDOM API name in root type ([eps1lon](https://github.com/eps1lon) in [#22363](https://github.com/facebook/react/pull/22363))
+* Add named hooks support to standalone and inline DevTools ([jstejada](https://github.com/jstejada) in [#22320](https://github.com/facebook/react/pull/22320) and [bvaughn](https://github.com/bvaughn) in [#22263](https://github.com/facebook/react/pull/22263))
```

Comment:
There are things I'd expect to see here that I don't...like showing errors and suspense names in the scheduling profiler.

Reply:
When I ran the release script locally, there were a lot more than 7 bullet points. The following should be added to this release:

#### Features
* Scheduling Profiler: Show Suspense resource .displayName ([USERNAME](https://github.com/USERNAME) in [#22451](https://github.com/facebook/react/pull/22451))
* * Scheduling Profiler marks should include thrown Errors ([USERNAME](https://github.com/USERNAME) in [#22419](https://github.com/facebook/react/pull/22419))
#### Bugfix
* DevTools encoding supports multibyte characters (e.g. "🟩") ([USERNAME](https://github.com/USERNAME) in [#22424](https://github.com/facebook/react/pull/22424))
#### Performance
* DevTools: Lazily parse indexed map sections ([USERNAME](https://github.com/USERNAME) in [#22415](https://github.com/facebook/react/pull/22415))
* DevTools: Hook names optimizations ([USERNAME](https://github.com/USERNAME) in [#22403](https://github.com/facebook/react/pull/22403))
* Replaced network.onRequestFinished() caching with network.getHAR() ([USERNAME](https://github.com/USERNAME) in [#22285](https://github.com/facebook/react/pull/22285))

Reply:
I'm curious, did you filter them out intentionally or did the sync script not log them for you?

Reply:
I filtered the performance ones out but I didn't see the feature or bugfix ones :O

Reply:
You didn't see them b'c you overlooked them or because the script didn't log them? If the latter, that's kind of concerning. (Can you do another test run from HEAD of `main` to confirm if the script is broken for you?)

In the short term, mind adding the ones above too? 🙇🏼

Reply:
Oooh I see it's because I didn't pull all the changes from main before running the release 😅 my bad! Maybe the script should do a pull from main too or remind you to do it

Reply:
Oooh 😮  You must have been pretty behind?

We should probably kill this PR then and restart fresh.